templates/OperationSnippet.stencil

```diff
@@ -15,6 +15,11 @@ public func {{ op.name }}(
     {% include "OperationMethodDecoding.stencil" %}
     {% include "OperationUrlSnippet.stencil" %}
     {% include "OperationQueryParamSnippet.stencil" %}
+    {% for param in op.params.query.optional %}
+    {% if param.source == "clientProperty" %}
+    {% include "OperationOptionalQueryParamSnippet.stencil" %}
+    {% endif %}
+    {% endfor %}
```

Comment:
Same here. This feels like we are modeling this wrong if this is what our stencil has too look like.

Reply:
In my earlier iteration, I create 2 arrays to separate these 2 scenarios: 
* op.params.query.optional
* op.params.query.optionalClientProperty
But this was reverted